src/Common.am

```diff
@@ -73,3 +79,19 @@ COMPAT_LIB = $(top_builddir)/compat/libcompatsquid.la $(LIBPROFILER)
 
 ## Some helpers are written in Perl and need the local shell defined properly
 subst_perlshell = sed -e 's,[@]PERL[@],$(PERL),g' <$(srcdir)/$@.pl.in >$@ || ($(RM) -f $@ ; exit 1)
+
+.h.hdrtest:
```

Comment:
Please consider moving this rule and some or all other additions in this Makefile.am into src/TestHeaders.am (that this file includes). AFAICT, doing so will improve Automake "code" encapsulation.

Reply:
done